### ISSUE TITLE
[本地部署] 删除最近一条对话失效

### ISSUE THREAD

**Issue by ackness** [GaiZhenbiao/ChuanhuChatGPT#152]
### 错误描述
> 删除最近一条对话失效, TypeError: delete_last_conversation() takes 4 positional arguments but 5 were given

### 复现操作
随便输入点东西问他 点删除

### 错误截图
> 
![image](https://user-images.githubusercontent.com/19150500/225210822-b0293778-64e4-4bcb-8f49-4ab78cf95b55.png)

### 终端（控制台）中的错误报告

```console
TypeError: delete_last_conversation() takes 4 positional arguments but 5 were given
```

#### 桌面系统

 - OS: MacOS
 - Browser: Chrome

#### 运行依赖

```shell
> pip show gradio
Name: gradio
Version: 3.20.1
> python --version
Python 3.9.16
```

**Comment by Keldos-Li** [comment]
这里没改掉？ @GaiZhenbiao 
https://github.com/GaiZhenbiao/ChuanhuChatGPT/blob/0bdeb882277dbf51c1f3342e58a6f7a4b69ff6f8/utils.py#L283-L298

**Comment by tuteng0915** [comment]
更倾向于是这里并不将model_select_dropdown作为输入
```
    delLastBtn.click(delete_last_conversation, [chatbot, history, token_count, use_streaming_checkbox, model_select_dropdown], [
                     chatbot, history, token_count, status_display], show_progress=True)
```
毕竟delete_last_conversation并没有做任何生成

**Comment by GaiZhenbiao** [comment]
修复啦

**Comment by Keldos-Li** [comment]
Resolved with 4389e9ff95101d82105aff53531cd817be34c9a3 and 6b8533672e12e670790f2594868fc943d4a85a08 ([Chuanhu](https://github.com/GaiZhenbiao)).
